### ISSUE TITLE
zig build configure? No such option!

### ISSUE THREAD

**Issue by tatumizer** [zigtools/zls#312]
I installed zls plugin for VSCode, it wants me to run "zig build configure", but there's no such command in zig 0.7.1
Any advice?

**Comment by InterplanetaryEngineer** [comment]
We recently changed this − you should now use `zls config` − but apparently did not update all documentation accordingly.

**Comment by tatumizer** [comment]
I downloaded zls from the release page (ver. 0.1.0, windows64), it doesn't recognize `zls config` either. Anyway, it's working somehow - though it shows syntax errors only AFTER I run `build`.  Still, it's quite helpful.

**Comment by InterplanetaryEngineer** [comment]
Yeah, the release build is now quite out of date. Until it is updated (soon, hopefully: https://github.com/zigtools/zls/issues/229#issuecomment-808930420), it seems like you will have to build it yourself or edit the zls.json config file manually.

ZLS does, however, generate live diagnostics by default. Maybe your editor is not set up to show them?

**Comment by Zooce** [comment]
I just built `zls` (from `master` as of now -- 759a7b7fd962ad4f4d5dc0fd5b21a33865aeaf59) with zig v0.8.0-dev.2215+fb8527b28, and I was still getting this error.

Turns out the `zls` executable is now located at `zig-out/bin/zls` instead of `zig-cache/bin/zls`.

**Comment by InterplanetaryEngineer** [comment]
Oof, that was a breaking change in Zig just a few days ago. Will update the docs.
